articles/relnotes/index.md

```diff
@@ -22,6 +25,24 @@ uid: microsoft.quantum.relnotes
 > * In order to work with a solution in Visual Studio, all projects contained in the solution need to be in the same folder as the solution or in one of its subfolders.  
 > * References between projects migrated to 0.6 and higher and projects using older package versions are **not** supported.
 
+# Version 0.8 (*PackageReference 0.8.1907.1701*)
+
+*Release date: July 12, 2019*
+
+This release contains the following:
+
+- New indexing locations for slicing arrays, [click here for more information](xref:microsoft.quantum.language.expressions#array-slices)
+- Added Dockerfile hosted on the [Microsoft Container Registry](https://github.com/microsoft/ContainerRegistry), [click here for more information](https://github.com/microsoft/iqsharp/blob/master/README.md)
```

Comment:
Strictly speaking, the Dockerfile isn't hosted on MCR, but defines how to build the image that we host on MCR. For SEO, I'd also suggest changing the text for the IQ# link:
```suggestion
- Added Dockerfile hosted on the [Microsoft Container Registry](https://github.com/microsoft/ContainerRegistry), see the [IQ# repository for more information](https://github.com/microsoft/iqsharp/blob/master/README.md)
```